### PR TITLE
Stack Print Method Added

### PR DESCRIPTION
**Pull request by AunSyedShah**
Print Stack Method Added

Example:
s.printStack();

### CODE REVIEW

File: lib/stack.dart [hunk 1]
```diff
@@ -30,4 +30,12 @@ class Stack<T> {
   int size() {
     return _list.length;
   }
+
+  // print stack
+  void printStack() {
```
Comment (ammaratef45) [inline lib/stack.dart:35]: @AunSyedShah I think we should name it print

File: lib/stack.dart [hunk 2]
```diff
@@ -30,4 +30,12 @@ class Stack<T> {
...
+    while (!_list.isEmpty) {
+      print(_list.last);
+      _list.removeLast();
```
Comment (ammaratef45) [inline lib/stack.dart:38]: @AunSyedShah Here, print will empty the stack

I think you should loop through it instead of that

**Review by ammaratef45 — CHANGES_REQUESTED** [review]
@AunSyedShah Thanks for the PR, I have concerns about a couple of things, please check my comments.

File: lib/stack.dart (hunk 2 above)
Comment (AunSyedShah) [reply lib/stack.dart:38]: I checked it, this implementation is working with cpp and dart. @ammaratef45. Do let me know with your kind suggestions further.

**Pull request closed without merging**